### PR TITLE
Altary Way

### PR DESCRIPTION
**Pull request by vtange**
**to IMC**
There seems to be a few soul-release altar quests in the QUEST200 file that are no longer used.
- one in **North Rudiziu Trail** about Merge attacking souls (this is replaced by getting the Scripture from one Pilgrim's Soul)
- <strike>another is one about cooling an angry red altar at **Oshima Highway**.</strike> found this one finally. It isn't one of the four-chain apparently

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -2303,7 +2303,7 @@ ETC_20150317_002302	{memo X}Succeeded in releasing the trapped soul!
 ETC_20150317_002303	{memo X}The soul is going to attack! Defeat them first!
 ETC_20150317_002304	{memo X}Succeeded in hunting the trapped monsters!
 ETC_20150317_002305	Flower
-ETC_20150317_002306	Souls which are restrained to the altar
+ETC_20150317_002306	$Soul restrained to the altar
```
Comment (Sunaries) [inline ETC.tsv]: I changed this to: Altar-bound Soul

File: ETC.tsv [hunk 2]
```diff
@@ -10060,7 +10060,7 @@ ETC_20150323_010061	$Don't lie! I will finish you too!
 ETC_20150323_010062	$You think I'll forgive you? 
 ETC_20150323_010063	$I will destroy you too!
 ETC_20150323_010064	$You think I'll fall for that? Come on and fight!
-ETC_20150323_010065	$Picking flower...
+ETC_20150323_010065	$Picking Flower...
```
Comment (Sunaries) [inline ETC.tsv]: I have this as: Picking a Flower

The original doesn't have the unnecessary extra periods.

File: ETC.tsv [hunk 3]
```diff
@@ -14490,11 +14490,11 @@ ETC_20150918_014491	$The fake Tree Root Crystal reduced your movement speed!
 ETC_20150918_014492	$The fake Tree Root Crystal froze you!
 ETC_20150918_014493	They betrayed me!
 ETC_20150918_014494	Looking through the piles of the leaves
-ETC_20150918_014495	The spirit is attacking!
-ETC_20150918_014496	The spirit of the pilgrim that was trapped inside the monster came out!{nl}Release the spirit!
-ETC_20150918_014497	Lure the monster and attack it to pull out the spirit!
-ETC_20150918_014498	To calm the restrained spirit, you would need a flower
-ETC_20150918_014499	The spirit of the pilgrim is attacking the Tree Root Crystal!
+ETC_20150918_014495	The soul is attacking!
+ETC_20150918_014496	The soul of the pilgrim that was trapped inside the monster came out!{nl}Release the soul!
```
Comment (Sunaries) [inline ETC.tsv]: The soul of the pilgrim that was trapped inside the monster came out!{nl}Release the soul! 

to

The pilgrim's soul that was trapped inside the monster came out!{nl}Release the pilgrim's soul!

File: ETC.tsv [hunk 4]
```diff
@@ -14490,11 +14490,11 @@ ETC_20150918_014491	$The fake Tree Root Crystal reduced your movement speed!
...
+ETC_20150918_014497	Lure the monster and attack it to pull out the soul!
+ETC_20150918_014498	To calm the restrained soul, you would need a flower
```
Comment (Sunaries) [inline ETC.tsv]: To calm the restrained soul, you would need a flower

to

You need flowers to calm the altar-bound soul

File: ETC.tsv [hunk 5]
```diff
@@ -14490,11 +14490,11 @@ ETC_20150918_014491	$The fake Tree Root Crystal reduced your movement speed!
...
+ETC_20150918_014499	The soul of the pilgrim is attacking the Tree Root Crystal!
```
Comment (Sunaries) [inline ETC.tsv]: The soul of the pilgrim is attacking the Tree Root Crystal!

to

The pilgrim's soul is attacking the Tree Root Crystal!

**Comment by vtange** [comment]
Good to merge

**Comment by imcgames** [comment]
:+1:

**Pull request merged**